tools/test_runner.js

```diff
@@ -127,6 +127,20 @@ Runner.prototype.run = function() {
     }
   }
 
+  var dependencies = this.test['dependencies'] || [];
+  var dependenciesFailed = [];
+  for (var i = 0, l = dependencies.length; i < l; ++i) {
```

Comment:
Can we remove/rename the `l` variable? It looks almost like a `1` (number one) and that can be a bit confusing at first glance.